docs/pages/identity-governance/entra-id/terraform.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title:  Configure the Entra ID Integration using Terraform
+description: Describes how to set up the Teleport Entra ID integration using Terraform
+sidebar_position: 3
+labels:
+ - how-to
+ - identity-governance
+ - terraform
+---
+
+This guide shows end-to-end Entra ID integration configuration using Terraform and `tctl`.
+You will configure Entra ID side of configuration using Terraform and install the Entra ID 
+plugin in Teleport using `tctl`.
+
+See [getting started with Entra ID integration](getting-started.mdx) for a guided end-to-end Entra ID configuration. 
+
+## Prerequisites
+
+- Teleport Identity Governance enabled for your Teleport cluster.
+- Your user must have privileged administrator permissions in Microsoft Azure and Entra ID tenant.
+
+## Step 1/3. Prepare Terraform
+
+First, set up `azuread` [provider](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs) in your 
+working environment from where you want to run the Terraform plan.
+
+Next, clone the Teleport Entra ID integration example module, which is available in the Teleport Github repository [TODO: update link placeholder](https://github.com/gravitational/teleport/pull/58252).
```

Comment:
Making sure the reminder is visible

Reply:
https://github.com/gravitational/teleport/pull/57835/commits/e43602727ec56e2d395fcbf3a93f30c1afdb1784